src/sql/workbench/contrib/notebook/browser/notebookViews/cellToolbar.css

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.nb-grid-stack .grid-stack-item.notebook-cell .actionbar {
```

Comment:
Will grid stack be implemented for all instances of an action bar? If so, you will not need to preface all these style selectors with `.nb-grid-stack`. Hard to tell exactly as there is no UI to test this on.

Reply:
It should only apply to the action bar used in the grid of the notebook views.